packages/scripts/utils/config.js

```diff
@@ -370,35 +363,44 @@ function getPhpFilePaths( context, props ) {
 	const srcDirectory = fromProjectRoot( context + sep );
 
 	return blockMetadataFiles.flatMap( ( blockMetadataFile ) => {
-		const blockJson = JSON.parse( readFileSync( blockMetadataFile ) );
-
 		const paths = [];
+		let parsedBlockJson;
+		try {
+			parsedBlockJson = JSON.parse( readFileSync( blockMetadataFile ) );
+		} catch ( error ) {
+			warn(
+				`Skipping scanning "${ blockMetadataFile.replace(
```

Comment:
"Skipping scanning" can probably be rephrased as it doesn't look right even for me while being non-native speaker 😅

Reply:
It's used in a couple of places. Maybe `Not scanning … due to …`.

Reply:
Done with https://github.com/WordPress/gutenberg/pull/66752/commits/c86a198f938a97329cbb1c0a2f666202225abe4a.